carbonserver/carbonserver/api/infra/repositories/repository_projects.py

```diff
@@ -36,6 +37,26 @@ def get_one_project(self, project_id):
             else:
                 return self.map_sql_to_schema(e)
 
+    def get_projects_from_team(self, team_id) -> List[Project]:
+        """Find the list of projects from a team in database and return it
+
+        :team_id: The id of the team to retreive projects from.
+        :returns: List of Projects in pyDantic BaseModel format.
+        :rtype: List[schemas.Project]
+        """
+        with self.session_factory() as session:
+            res = session.query(SqlModelProject).filter(
+                SqlModelProject.team_id == team_id
+            )
+            if res.first() is None:
+                return []
+            else:
+                projects = []
```

Comment:
You could have write it in one line:
return [self.map_sql_to_schema(e) for e in res]